core/src/modules/errors.ts

```diff
@@ -17,6 +17,7 @@ export namespace Errors {
     constructor(message: string, code = "AUTHENTICATION_ERROR") {
       super(message);
       this.code = code;
+      delete this.stack;
```

Comment:
Also in this PR: I'm cleaning up our logs when we hit an `AUTHENTICATION_ERROR` - the stack trace is not meaningful and the action log tells you enough:

Before:

```
2022-01-27T02:25:56.460Z - error: [ action @ web ] to=127.0.0.1 action=destinations:list params={"action":"destinations:list","apiVersion":"1","modelId":"people","serverToken":"change_me"} duration=20 method=GET pathname=/api/v1/destinations error=Error: Please log in to continue code=AUTHENTICATION_ERROR stack=Error: Please log in to continue
    at cls_1.CLS.wrap.catchError (/Users/evan/workspace/grouparoo/grouparoo/core/src/modules/middleware/authentication.ts:75:17)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async /Users/evan/workspace/grouparoo/grouparoo/core/src/modules/cls.ts:87:21
    at async /Users/evan/workspace/grouparoo/grouparoo/node_modules/.pnpm/sequelize@6.11.0_98f032adade92d92d5295670c0365597/node_modules/sequelize/lib/sequelize.js:1097:24
    at async wrapInternal (/Users/evan/workspace/grouparoo/grouparoo/core/src/modules/cls.ts:86:5)
    at async Object.CLS.wrap (/Users/evan/workspace/grouparoo/grouparoo/core/src/modules/cls.ts:48:27)
    at async authenticateTeamMemberFromSession (/Users/evan/workspace/grouparoo/grouparoo/core/src/modules/middleware/authentication.ts:63:10)
    at async authenticateTeamMember (/Users/evan/workspace/grouparoo/grouparoo/core/src/modules/middleware/authentication.ts:144:13)
    at async ActionProcessor.preProcessAction (/Users/evan/workspace/grouparoo/grouparoo/node_modules/.pnpm/actionhero@28.1.4/node_modules/actionhero/dist/classes/actionProcessor.js:162:17)
    at async ActionProcessor.runAction (/Users/evan/workspace/grouparoo/grouparoo/node_modules/.pnpm/actionhero@28.1.4/node_modules/actionhero/dist/classes/actionProcessor.js:331:40)
    at async WebServer.processAction (/Users/evan/workspace/grouparoo/grouparoo/node_modules/.pnpm/actionhero@28.1.4/node_modules/actionhero/dist/classes/server.js:113:22)
```

After:

```
2022-01-27T02:25:20.916Z - error: [ action @ web ] to=127.0.0.1 action=destinations:list params={"action":"destinations:list","apiVersion":"1","modelId":"people","serverToken":"change_me"} duration=17 method=GET pathname=/api/v1/destinations error=Error: Please log in to continue code=AUTHENTICATION_ERROR
```